telemetry/specs/INT.mdk

```diff
@@ -879,7 +879,7 @@ hop-by-hop INT header must fit in a single Geneve option.
 In this section, we define the format for INT hop-by-hop metadata headers,
 and the metadata itself.
 
-INT Metadata Header and Metadata Stack:
+INT Metadata Header and Metadata Stack (Version = 1):
```

Comment:
I think we decided to keep just v2 contents, instead of keeping both v1 and v2 in the same version of spec. We will have all previous versions of the specs in the github and p4.org spec link.

Reply:
Fixed.